taskt/Core/Automation/Commands/Web Browser Commands/GetHTMLSourceCommand.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
+
+namespace taskt.Core.Automation.Commands
+{
+    [Serializable]
+    [Group("Web Browser Commands")]
+    [Description("This command downloads the HTML source of a web page for parsing without using browser automation.")]
+
+    public class GetHTMLSourceCommand : ScriptCommand
+    {
+        [XmlAttribute]
+        [PropertyDescription("URL")]
+        [InputSpecification("Enter a valid URL that you want to collect data from.")]
+        [SampleUsage("http://mycompany.com/news || {vCompany}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public string v_WebRequestURL { get; set; }
+
+        [XmlAttribute]
+        [PropertyDescription("Execute Request As Logged On User")]
+        [PropertyUISelectionOption("Yes")]
+        [PropertyUISelectionOption("No")]
+        [InputSpecification("Sets currently logged on user authentication information for the request.")]
+        [SampleUsage("")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
```

Comment:
We might not need the UIHelper for the drop down.
Please remove.

Reply:
good catch. done